### DISCUSSION TITLE
Table ShowCopyColumn 功能不应该复制明细行内容

### DISCUSSION THREAD

**Discussion by sundows** [dotnetcore/BootstrapBlazor#4369] [Q&A] [answered]
### Which class is this unit test associated with?

https://github.com/user-attachments/assets/05559ef7-ae9e-4bb1-b1ad-1a30fe69f81a
![24}XQ0I`_HZZ@$48`WYXN$S](https://github.com/user-attachments/assets/a79a40d2-020b-45aa-bf89-6d5254482ef4)

在使用 Table 组件的复制列名（ShowCopyColumn）的功能时，发现一个现象（详看上面附加的视频），如果父行有明细行，展开明细行后，再使用父行的复制列名的功能时，会把明细行的内容也复制到剪贴板中，不知这是不是正常的现象？

**Comment by bb-auto** [comment]
@sundows Thank you for submitting. We will give feedback later.

**Comment by ArgoZhang** [comment]
@sundows 感谢提交问题，应该是 bug 不应该复制明细行的内容

**Comment by ArgoZhang** [comment, marked as answer]
@sundows 经过测试目前程序就是不会拷贝明细行内的数据的

**Comment by sundows** [comment]
@ArgoZhang 该问题仍然在 V8.10.2 中存在。
测试工程项目：在 Github 下载的 BootstrapBlazor.Server V8.10.2 工程。
修改文件：TablesDetailRow.razor
修改内容：只选择表格的的“Name”字段增加 ShowCopyColumn="true" 其它内容不作任何修改，如下图所示：
![image](https://github.com/user-attachments/assets/8804d807-4507-438d-aee9-6db19d42434d)

**不会出现复制明细行内容的情况：**

- 明细行为 _单列_ 表格，如“简单应用”示例或“固定表头开启明细行功能”示例中的表格。

**会出现复制明细行内容的情况：**

- 明细行为 _多列_ 表格，如“嵌套 Table 组件应用”示例或“明细行中使用 Tab 组件”示例中的表格。

**Comment by ArgoZhang** [comment]
@sundows 建议按照 Issue 规范提供复现工程，请换位思考一下。每个人都这样发个图，打点字，你们一个人需要1分钟，我们就需要读你们的话，按照你们图来改，我们需要翻倍的时间来完成最简单的复现这第一个步骤
